etl/steps/data/garden/countries/2023-09-25/isd.meta.yml

```diff
@@ -0,0 +1,164 @@
+# NOTE: To learn more about the fields, hover over their names.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/dataset/
+dataset:
+  update_period_days: 1400
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/tables/
+tables:
+  isd:
+    variables:
+      cowid:
+        title: COW ID
+        description_from_producer: |-
+          A three-letter abbreviation given by the COW dataset. For new cases, a COW ID was created.
+      statename:
+        title: State Name
+        description_from_producer: |-
+          The name used for the state in the ISD dataset.
+      othername:
+        title: Other Name
+        description_from_producer: |-
+          Other names by which the state is known
+      isd_category:
+        title: ISD Category
+        description_from_producer: |-
+          Category 1 states have same start and end years as ISD V1. Category 2 states were in ISD V1 but have changed start and/or end years. Category 3 states are new states that were not in ISD V1.
+      region:
+        title: Region
+        description_from_producer: |-
+          The COW index number for the geographic region in which the state is located.
+      start_am:
+        title: Start-Am
```

Comment:
Some titles like this one, the next two or others are not so clear for people not creating the dataset